test/core/sup_tree_core/reduction_log_writer_test.exs

```diff
@@ -0,0 +1,11 @@
+defmodule AntikytheraCore.ReductionLogWriterTest do
+  use Croma.TestCase
+
+  @dir Path.join([__DIR__, "..", "..", "..", "_build", "test", "log", "antikythera"]) |> Path.expand()
+
+  test "should exists" do
+    assert Process.whereis(AntikytheraCore.ReductionLogWriter) != nil
```

Comment:
As long as you use [`Croma.TestCase`](https://hexdocs.pm/croma/Croma.TestCase.html), the module under test is automatically aliased.
```suggestion
    assert Process.whereis(ReductionLogWriter) != nil
```